com.unity.render-pipelines.core/Editor/Camera/CameraUI.Drawers.cs

```diff
@@ -23,6 +23,8 @@ public enum Expandable
             Rendering = 1 << 5,
```

Comment:
com.unity.render-pipelines.core/Editor/Camera/CameraUI.Drawers.cs.meta is missing